tests/correctness/numeric-types/fixed-point/binary-operators.expect

```diff
@@ -8,6 +8,12 @@
   "signed_add": [
     "0.5"
   ],
+  "signed_div": [
+    "-0.5"
```

Comment:
Why are the values for signed and unsigned `div` different? Are the bits in the fixed point representations being interpreted differently?

Reply:
For a sanity check, I use different values for signed and unsigned binary operators:

https://github.com/cucapra/calyx/blob/4862208056e54118eafc472abc068a44279f1b97/tests/correctness/numeric-types/fixed-point/binary-operators.futil.data#L1-L45

Reply:
Got it!